src/NuGet.Core/NuGet.Packaging/Signing/Utility/CertificateUtility.cs

```diff
@@ -29,19 +29,28 @@ public static class CertificateUtility
         public static string X509Certificate2ToString(X509Certificate2 cert, HashAlgorithmName fingerprintAlgorithm)
         {
             var certStringBuilder = new StringBuilder();
-            X509Certificate2ToString(cert, certStringBuilder, fingerprintAlgorithm, indentation: "  ");
+            X509Certificate2ToString(cert, certStringBuilder, fingerprintAlgorithm, indentation: 1);
             return certStringBuilder.ToString();
         }
 
-        private static void X509Certificate2ToString(X509Certificate2 cert, StringBuilder certStringBuilder, HashAlgorithmName fingerprintAlgorithm, string indentation)
+        private static void X509Certificate2ToString(X509Certificate2 cert, StringBuilder certStringBuilder, HashAlgorithmName fingerprintAlgorithm, int indentation)
         {
+            void AppendLine(string str)
```

Comment:
Why not use [`StringBuilder.Insert(int, string, int)`](https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder.insert?view=netstandard-2.1#System_Text_StringBuilder_Insert_System_Int32_System_String_System_Int32_)?